src/HomePage.js

```diff
@@ -13,7 +18,69 @@ class HomePage extends React.Component {
   render() {
     return (
       <div>
-        
+        <NavBar></NavBar>
+        <div class='card_list'>
+            <div class ='card text-black bg-info' style={{ width: '18rem' }}>
+              <Card.Img src="animu_logo.png" alt="Logo" />
+              <Card.Body>
+                <Card.Title><p class='title'>About me</p></Card.Title>
+                <Card.Text>
+                  <Container>
+                    <p class='info_grid'>
+                      <p class='attributeName redfont'>Name : </p>
+                      <p class='attributeValue'>Binaru Appuhamy</p>
+                    </p>
+                    <p class='info_grid'>
+                      <p class='attributeName'>Current year : </p>
+                      <p class='attributeValue'>ECE PEY</p>
+                    </p>
+                    <p class='info_grid'>  
+                      <p class='attributeName'>Hobbies : </p>
+                      <p class='attributeValue'>Read manga and watch anime!</p>
+                    </p>
+                  </Container>
+                </Card.Text>
+                <Accordion flush>
+                  <Accordion.Item eventKey="0">
+                    <Accordion.Header>More Hobbies</Accordion.Header>
+                    <Accordion.Body>
+                      Nope nothin here.
+                    </Accordion.Body>
+                  </Accordion.Item>
+                </Accordion>
+              </Card.Body>
+            </div>
+            <div class ='card text-black bg-info' style={{ width: '18rem' }}>
+              <Card.Img class = "circle_crop" src="animu_logo.png" alt="Logo" />
+              <Card.Body>
+                <Card.Title><p class='title'>About me</p></Card.Title>
+                <Card.Text>
+                  <Container>
+                    <p class='info_grid'>
+                      <p class='attributeName'>Name : </p>
+                      <p class='attributeValue'>Binaru Appuhamy</p>
+                    </p>
+                    <p class='info_grid'>
+                      <p class='attributeName'>Current year : </p>
```

Comment:
![image](https://user-images.githubusercontent.com/40864187/155869521-6366b0c5-dcd3-4e26-a600-2f678b7e4032.png)

Is this a screen resolution issue...cause I see it fine on my end as shown in the image above...if possible can you elaborate a bit about this issue.

Reply:
![Screen Shot 2022-02-27 at 12 34 23 AM](https://user-images.githubusercontent.com/49298813/155869626-40a9b2de-bf44-4eaf-92f8-eb937ca81e1e.png)

Hmm that's weird because it looks like this on my end

Reply:
![Screen Shot 2022-02-27 at 12 37 09 AM](https://user-images.githubusercontent.com/49298813/155869675-84b7aa7f-175f-4c8a-a1fd-6bc3c049508b.png)
It is a font issue. I tried adding this in HomePage.css:
```
div {
    font-family:'Times New Roman', Times, serif;
}
```
Now it looks fine